### ISSUE TITLE
selective match loss

### ISSUE THREAD

**Issue by quanfeifan** [iSEE-Laboratory/EconomicGrasp#4]
Hi, I'm interested of the selective match loss in your paper, and i cannot find the corresponding code in this repo, could u point it out?
您好，我对于selective match loss比较感兴趣，然而我并没有在代码中发现相关的实现，您可以给我指个路吗？
Thanks!
感谢！

**Comment by Fleshmannnnnn** [comment]
> Hi, I'm interested of the selective match loss in your paper, and i cannot find the corresponding code in this repo, could u point it out?你好，我对你论文中的选择性匹配丢失感兴趣，但我在这个repo中找不到相应的代码，你能指出吗？ 您好，我对于selective match loss比较感兴趣，然而我并没有在代码中发现相关的实现，您可以给我指个路吗？ Thanks! 谢谢！ 感谢！
/utils/label_generation.py：
#only compute loss in the points with correct matching (so compute the mask first)
        dist = compute_pointwise_dists(seed_xyz, grasp_points_merged)
        valid_point_mask = dist < 0.005
        valid_view_mask = torch.zeros(num_samples, dtype=torch.bool).to(seed_xyz.device)
        valid_view_mask[pid] = True
        valid_points_count = valid_points_count + torch.sum(valid_point_mask)
        valid_views_count = valid_views_count + torch.sum(valid_view_mask)
        valid_mask = valid_point_mask & valid_view_mask

**Comment by DravenALG** [comment]
First we compute the mask in "utils/label_generation.py" (line 151-158), then we use the computed mask in "models/loss_economicgrasp.py" (line 77, 93, 108, 123).
